resources/github-comment-markdown.template

```diff
@@ -50,6 +50,15 @@
 | Total        | ${(testsSummary?.total) ?: 0}   |
 <%}%>
 
+#### Trends :test_tube:
+
+<% if(env?.JOB_URL != null && !env?.JOB_URL.equals('null')) {%>
+![Image of Build Times](${env?.JOB_URL}buildTimeGraph/png)
```

Comment:
For instance https://apm-ci.elastic.co/job/apm-shared/job/apm-pipeline-library-mbp/job/master/buildTimeGraph/png